Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:18-alpine
+
+WORKDIR /app
+COPY . .
+
+RUN apk add --no-cache --virtual .build-deps alpine-sdk python3 git openjdk8-jre cmake
```

Comment:
You may not need these build dependencies, IIRC the jdk was required for the OpenAPI generator package, and the native compilation tools were used for some npm packages that also aren't used here

Reply:
Done